src/app_service/service/community/dto/community.nim

```diff
@@ -87,13 +95,13 @@ proc toChat(jsonObj: JsonNode): Chat =
   discard jsonObj.getProp("position", result.position)
   discard jsonObj.getProp("categoryID", result.categoryId)
 
-proc toCategory(jsonObj: JsonNode): Category =
+proc toCategory*(jsonObj: JsonNode): Category =
   result = Category()
-  discard jsonObj.getProp("id", result.id)
+  discard jsonObj.getProp("category_id", result.id)
```

Comment:
@jrainville is it safe to do this change? Does this affect somehow response of the rpc `joinedCommunities`?

Reply:
Turns out we were both right. We receive both types from status-go depending on the rpc method, so I updated it to support both